Project/src/main/java/com/github/lgooddatepicker/components/TimePicker.java

```diff
@@ -994,7 +994,7 @@ private void zInternalSetLastValidTimeAndNotifyListeners(LocalTime newTime) {
      */
     private void zInternalSetTimeTextField(String text) {
         skipTextFieldChangedFunctionWhileTrue = true;
-        if (settings.useLowercaseForDisplayTime) {
+        if (settings.useLowercaseForDisplayTime && text != null) {
```

Comment:
great catch with the NPE here :+1:

Reply:
Could you maybe set the setting `settings.useLowercaseForDisplayTime` explicitly to `true` when testing for this NPE, so that we don't loose test coverage in case someone changes the default of that setting to `false`? Thanks!